.github/workflows/docker_production_image.yml

```diff
@@ -0,0 +1,45 @@
+name: One App Production Image Build and Push
+on:
+  repository_dispatch:
+    types: [one-app-docker-prod-build]
+jobs:
+  one-app-docker-prod-build:
+    name: One App Production Image Build and Push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker login
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login --username ${{ secrets.DOCKER_USER }} --password-stdin
+      - name: Build production docker image
+        run: docker build -t prod .
+      - name: Extract sem versions
+        id: get_sem_version
+        run: |
+            MAJOR="$(echo ${{ github.event.client_payload.docker_tag_version}} | cut -d'.' -f1)"
+            MINOR="$(echo ${{ github.event.client_payload.docker_tag_version}} | cut -d'.' -f2)"
+            PATCH="$(echo ${{ github.event.client_payload.docker_tag_version}} | cut -d'.' -f3)"
+            echo ::set-output name=major_version::${MAJOR}
+            echo ::set-output name=minor_version::${MINOR}
+      - name: Tag Docker Images
+        run: |
+          docker tag prod ${{ secrets.DOCKER_USER }}/one-app:${{ steps.get_sem_version.outputs.major_version}}
+          docker tag prod ${{ secrets.DOCKER_USER }}/one-app:${{ steps.get_sem_version.outputs.major_version}}.${{ steps.get_sem_version.outputs.minor_version}}
+          docker tag prod ${{ secrets.DOCKER_USER }}/one-app:${{ github.event.client_payload.docker_tag_version}}
+          docker tag prod ${{ secrets.DOCKER_USER }}/one-app:latest
+      - name: Push Docker Images
+        run: |
+          docker push ${{ secrets.DOCKER_USER }}/one-app:${{ steps.get_sem_version.outputs.major_version}}
+          docker push ${{ secrets.DOCKER_USER }}/one-app:${{ steps.get_sem_version.outputs.major_version}}.${{ steps.get_sem_version.outputs.minor_version}}
+          docker push ${{ secrets.DOCKER_USER }}/one-app:${{ github.event.client_payload.docker_tag_version}}
+          docker push ${{ secrets.DOCKER_USER }}/one-app:latest
+      - name: Trigger Image Publishing
```

Comment:
are we able to have this triggered from the `docker_tag_check` action ?

Reply:
yes that is what is happening the docker tag check triggers this action to start.